### DISCUSSION TITLE
Secure

### DISCUSSION THREAD

**Discussion by awptechnologies** [leikoilja/ha-google-home#826] [General]
When using docker container to generate master token is it safe? there is no data stored and sent somewhere is there?

**Comment by leikoilja** [comment]
Hey, @awptechnologies. This is secure following how docker technology works.
You are loading docker image first (generic image) and then it starts a one-time lived container where you request tokens and print them on the screen after obtaining those. After the script is done - container is removed together will all it's history. Nothing is stored or send anywhere
